packages/flutter/test/semantics/semantics_service_test.dart

```diff
@@ -6,10 +6,15 @@ import 'dart:ui' show TextDirection;
 
 import 'package:flutter/semantics.dart';
 import 'package:flutter/services.dart' show SystemChannels;
+import 'package:flutter_test/flutter_test.dart' show TestWidgetsFlutterBinding;
 
 import '../flutter_test_alternative.dart';
 
 void main() {
+  setUp(() {
```

Comment:
Since this is a static call, it can just be done in `main()` directly vs in `setUp()`

Reply:
Same applies to the tests below

Reply:
Done for all.